src/components/admin/AddContent.tsx

```diff
@@ -2,43 +2,136 @@
 import { useEffect, useState } from 'react';
 import { AddNotionMetadata } from './AddNotionMetadata';
 import { Input } from '../ui/input';
+import { useRecoilState } from 'recoil';
+import { loader } from '@/store/atoms/loader';
+import DiscordService from '@/app/services/DiscordService';
+import { Checkbox } from '../ui/checkbox';
 
 export const AddContent = ({
+  rest,
   courseId,
   parentContentId,
+  courseTitle,
 }: {
+  rest: string[];
   courseId: number;
   parentContentId?: number;
+  courseTitle: string;
 }) => {
   const [type, setType] = useState('folder');
   const [imageUri, setImageUri] = useState('');
   const [title, setTitle] = useState('');
   const [metadata, setMetadata] = useState({});
+  const [checked, setChecked] = useState<Boolean>(true);
   const [adminPassword, setAdminPassword] = useState('');
+  const [loading, setLoading] = useRecoilState(loader);
+
+  const { sendUpdateToDiscord } = DiscordService();
+
+  interface DiscordData {
+    type: string;
+    thumbnail: string;
+    title: string;
+    courseTitle: string;
+    courseId: number;
+    currFolderId: number;
+    mediaId: number;
+  }
+
+  const postOnDiscord = (data: DiscordData) => {
+    sendUpdateToDiscord(data);
+  };
+
+  const handleContentSubmit = async () => {
+    setLoading(true);
+    const response = await fetch('/api/admin/content', {
+      body: JSON.stringify({
+        type,
+        description: '',
+        thumbnail: imageUri,
+        title,
+        courseId,
+        parentContentId,
+        metadata,
+        adminPassword,
+      }),
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    setLoading(false);
+
+    const respData: { id: number } = await response.json();
+
+    const data = {
```

Comment:
This is unnecessary here, do it server side here  at /api/admin/content

Reply:
so @siinghd , do you want me to shift all of this logic to api/admin/content route and send checked variable in this request ?
````
const response = await fetch('/api/admin/content', {
      body: JSON.stringify({
        type,
        description: '',
        thumbnail: imageUri,
        title,
        courseId,
        parentContentId,
        metadata,
        adminPassword,
      }),
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
      },
    });```

Reply:
Yes sir, also if no webhook env is set, do not send the dicord request

Reply:
Okay ser , understood . 🙌

Reply:
done ser ✌